README.md

```diff
@@ -55,7 +76,23 @@ if __name__ == "__main__":
 
 ### Modularity
 * If you have cut & paste code in two different places, it probably should be a
-  method
+  method (or maybe a loop)
```

Comment:
functions are objects and you can get attributes by string. Ok the level of loops I do shouldn't be included here.

Reply:
I'm not sure what this comment is saying/asking for?

Reply:
It's a non-sequitur and not actionable. You just made me think of how sometimes I do things like:

``` python
for convertor func in {float, int}:
    pass

for attr_name in {"number", "density"}:
    attribute = getattr(obj, attr_name)
    pass
```